tests/core/test_actions.py

```diff
@@ -344,6 +367,164 @@ async def test_remote_action_with_template_param(
     ]
 
 
```

Comment:
can we please add a test that makes sure that every event type is covered by the jsonschema? [rasa.shared.utils.common.all_subclasses](https://github.com/RasaHQ/rasa/blob/74ce55924e820be3cd81fcc105ed4ba6b7225df4/tests/shared/core/test_events.py#L292) should helpful here

Reply:
This was super helpful - I found two subclasses that are not covered: `rasa.core.test.WronglyPredictedAction` and `rasa.core.test.WronglyClassifiedUserUtterance` - I can add these, although they reside in a different module. Please let me know if I should ignore them?

Reply:
mhm, these can actually be ignored 👍🏻 They are only used for testing and will never enter from the outside / leave Rasa Open Source